cms/djangoapps/contentstore/views/course.py

```diff
@@ -328,7 +328,8 @@ def _course_outline_json(request, course_module):
         course_module,
         include_child_info=True,
         course_outline=True,
-        include_children_predicate=lambda xblock: not xblock.category == 'vertical'
+        include_children_predicate=lambda xblock: not xblock.category == 'vertical',
+        user=request.user
```

Comment:
BTW: if you end this line with a comma, then the next person to add another line won't have to change it, as you had to change the previous line.